### PR TITLE
Add support to use ssh key in torpedo deploy

### PR DESCRIPTION
**Pull request by thiguetta**

### CODE REVIEW

File: drivers/node/ssh/ssh.go [hunk 1]
```diff
@@ -43,11 +44,13 @@ func getKeyFile(keypath string) (ssh_pkg.Signer, error) {
 	file := keypath
 	buf, err := ioutil.ReadFile(file)
 	if err != nil {
+		logrus.Errorf("err: %s", err.Error())
```
Comment (harsh-px) [inline drivers/node/ssh/ssh.go]: print detail in the log .. something like Failed to read key file due to...

File: drivers/scheduler/k8s/specs/elasticsearch/px-elasticdata-app.yaml [hunk 2]
```diff
@@ -71,6 +71,8 @@ spec:
           value: "false"
         - name: "ES_JAVA_OPTS"
           value: "-Xms256m -Xmx256m"
+        - name: "NETWORK_HOST"
+          value: "_eth0_"
```
Comment (harsh-px) [inline drivers/scheduler/k8s/specs/elasticsearch/px-elasticdata-app.yaml]: do you have a reference online that suggests to use this?

Comment (thiguetta) [reply drivers/scheduler/k8s/specs/elasticsearch/px-elasticdata-app.yaml]: I found it here and worked for our case https://github.com/pires/kubernetes-elasticsearch-cluster/issues/64

File: drivers/node/ssh/ssh.go (hunk 1 above)
Comment (harsh-px) [reply drivers/node/ssh/ssh.go]: @thiguetta ^^

File: drivers/node/ssh/ssh.go [hunk 3]
```diff
@@ -43,11 +44,13 @@ func getKeyFile(keypath string) (ssh_pkg.Signer, error) {
 	file := keypath
 	buf, err := ioutil.ReadFile(file)
 	if err != nil {
+		logrus.Errorf("failed to reade ssh key file. Cause: %s", err.Error())
```
Comment (harsh-px) [inline drivers/node/ssh/ssh.go]: typo for `reade`.

**Pull request merged**